src/PKSim.Core/Snapshots/Mappers/AdvancedParameterMapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using ModelAdvancedParameter = PKSim.Core.Model.AdvancedParameter;
+using SnapshotAdvancedParameter = PKSim.Core.Snapshots.AdvancedParameter;
+
+namespace PKSim.Core.Snapshots.Mappers
+{
+   public class AdvancedParameterMapper : ParameterContainerSnapshotMapperBase<ModelAdvancedParameter, SnapshotAdvancedParameter>
+   {
+      public AdvancedParameterMapper(ParameterMapper parameterMapper) : base(parameterMapper)
+      {
+      }
+
+      public override SnapshotAdvancedParameter MapToSnapshot(ModelAdvancedParameter advancedParameter)
+      {
+         return SnapshotFrom(advancedParameter, snapshot =>
+         {
+            //the parameter path is what identified the advanced parameter. The name is not used anywhere. We will use the path as key
```

Comment:
What do you mean by 'path as key'?
Are you using the Name property here to make the json more easy to understand? As in, during deserialization, will it be used?